examples/js/web-worker/service-worker.js

```diff
@@ -0,0 +1,16 @@
+import Bugsnag from '/node_modules/@bugsnag/web-worker/dist/notifier.js'
+Bugsnag.start('ed0820ec954fcc35c3bc4f18fd36b206')
```

Comment:
is this a real api key?

Reply:
Yeah, I kept mine in there for testing purposes but will keep this PR as close to the real thing as possible from now on. I'll just replace it locally when I do the final tests.